docs/advanced-features/security-headers.md

```diff
@@ -43,7 +43,7 @@ This header controls DNS prefetching, allowing browsers to proactively perform d
 
 This header informs browsers it should only be accessed using HTTPS, instead of using HTTP. Using the configuration below, all present and future subdomains will use HTTPS for a `max-age` of 2 years. This blocks access to pages or subdomains that can only be served over HTTP.
 
-If you're deploying to [Vercel](https://vercel.com/docs/edge-network/headers#strict-transport-security), this header is not necessary as it's automatically added to all deployments unless you declare headers in your [`next.config.js`](https://nextjs.org/docs/api-reference/next.config.js/introduction)`.
+If you're deploying to [Vercel](https://vercel.com/docs/edge-network/headers#strict-transport-security), this header is not necessary as it's automatically added to all deployments unless you declare [`headers`](https://nextjs.org/docs/api-reference/next.config.js/headers) in your `next.config.js`.
```

Comment:
@styfle  I thought about this at first, but it was essentially a duplicate link for the page.

Reply:
https://www.youtube.com/watch?v=5AsLWIuNNMU old video to my understanding two links for the same page would boost the links page rank for the page priority making this the most important link which may be fine. Appreciate the suggestions on the pr 🙌.

Reply:
Seems correct to me since this page is about headers afterall 👍